.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,36 @@
+name: GitHub Pages
+
+on:
+    push:
```

Comment:
You should rebase (`git rebase origin/master && git push --force-with-lease`) and run `pre-commit run --all-files`.